x-pack/platform/plugins/shared/saved_objects_tagging/public/utils.ts

```diff
@@ -5,63 +5,12 @@
  * 2.0.
  */
 
-import type { SavedObject, SavedObjectReference } from '@kbn/core/types';
-import type { SavedObjectsFindOptionsReference } from '@kbn/core/public';
 import type { Tag } from '../common/types';
-import { tagSavedObjectTypeName } from '../common';
-
-type SavedObjectReferenceLike = SavedObjectReference | SavedObjectsFindOptionsReference;
-
-export {
-  tagIdToReference,
-  replaceTagReferences as updateTagsReferences,
-} from '../common/references';
-
-export const getObjectTags = (
```

Comment:
Moved to common. See https://github.com/elastic/kibana/pull/207010/files#r1919269076.